packages/mask/src/extension/popups/pages/Wallet/CreatePassword/index.tsx

```diff
@@ -0,0 +1,151 @@
+        padding: 16,
+        position: 'fixed',
+        bottom: 0,
+        width: '100%',
+        background: 'rgba(255, 255, 255, 0.8)',
+        boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.05)',
+        backdropFilter: 'blur(8px)',
+    },
+    button: {
+        fontWeight: 600,
+        padding: '11px 10px',
+        borderRadius: 8,
+        backgroundColor: '#07101B',
+        fontSize: 14,
+        lineHeight: '18px',
+    },
+    disabled: {
+        backgroundColor: '#B5B7BB!important',
+        color: '#ffffff!important',
+    },
+}))
+
+const CreatePassword = memo(() => {
+    const { t } = useI18N()
+    const { classes } = useStyles()
+    const navigate = useNavigate()
+    const {
+        control,
+        handleSubmit,
+        setError,
+        formState: { errors, isValid },
+        schema,
+    } = usePasswordForm()
+
+    const [{ loading }, onConfirm] = useAsyncFn(
+        async (data: zod.infer<typeof schema>) => {
+            try {
+                await WalletRPC.setPassword(data.password)
+                navigate(PopupRoutes.ImportWallet, { replace: true })
+            } catch (error) {
+                if (error instanceof Error) {
+                    setError('password', { message: error.message })
+                }
+            }
+        },
+        [history, setError],
+    )
+
+    const onSubmit = handleSubmit(onConfirm)
+
+    return (
+        <>
+            <Box className={classes.header}>
+                <Icons.MaskWallet size={64} />
+            </Box>
+            <Typography className={classes.title}>{t('popups_wallet_create_payment_password')}</Typography>
+            <Typography className={classes.tip}>{t('popups_wallet_create_payment_password_tip')}</Typography>
+            <form className={classes.form}>
+                <Controller
+                    control={control}
+                    render={({ field }) => (
+                        <PasswordField
+                            {...field}
+                            type="password"
+                            variant="filled"
+                            placeholder={t('popups_wallet_payment_password')}
+                            error={!isValid && !!errors.password?.message}
+                            helperText={!isValid ? errors.password?.message : ''}
+                        />
+                    )}
+                    name="password"
+                />
+                <Controller
+                    render={({ field }) => (
+                        <PasswordField
+                            {...field}
+                            error={!isValid && !!errors.confirm?.message}
+                            helperText={!isValid ? errors.confirm?.message : ''}
+                            type="password"
+                            variant="filled"
+                            placeholder={t('popups_wallet_re_payment_password')}
+                        />
+                    )}
+                    name="confirm"
```

Comment:
```suggestion
                    name="password"
```

Reply:
The re-confirm password's field name is confirm